CHANGES.md

```diff
@@ -1,6 +1,10 @@
 ## Changelog
 
-### 1.71 (2018-11-05)
+### 1.72
```

Comment:
nit: date?

Reply:
we add a date after it gets released (which will likely happen some time today)
So next time, when add comments for 1.73, will add data for 1.72